source/includes/_upload.md

```diff
@@ -77,63 +97,81 @@ curl "https://siasky.net/skynet/skyfile" -F files[]=@./images/image1.png -F file
 
 ```python
 from siaskynet import Skynet
-response = Skynet.upload_directory("./images")
-print(responsek)
+
+url = Skynet.upload_directory("./images")
+print("Upload successful, url: " + url)
```

Comment:
This, I think, is an example of a discrepancy we want to iron out of the SDKs. Why would uploading a directory return a `url` while uploading a file returns a `skylink`. I'm also slightly surprised we are returning a `url`...

It we currently return a `url` instead of a `skylink` then this is fine. But we should make note that this is a change we need to make to all SDK libs. Perhaps start a TODO somewhere or an issue where we list all of these minor changes. It wouldn't hurt to see an overview of all the SDKs and the functions they support alongside their interface.

Reply:
Wrote up a couple of issues with your points. Getting started on mapping out the SDKs now.